src/app/modernizing/modernizing.component.html

```diff
@@ -69,76 +70,77 @@ <h1 tabindex="-1">Land Use Planning</h1>
         <p>
           B.C.’s public service is committed to anti-racism and diversity, equity, and
           inclusion strategies to ensure all processes are free from discrimination.
-           Racist or discriminatory comments received by planning teams throughout
-           land use planning engagement periods will not be tolerated nor considered
-            in the final planning outcomes.
+          Racist or discriminatory comments received by planning teams throughout
+          land use planning engagement periods will not be tolerated nor considered
+          in the final planning outcomes.
         </p>
         <p>
-          Visit the B.C. 
-            <a href="https://www2.gov.bc.ca/gov/content/industry/crown-land-water/land-use-planning">
-              government’s land use planning program homepage
-            </a>
-          to learn more about provincial land use planning policy and guidance, completed provincial land use plans and legal direction, and spatial data.
+          Visit the B.C.
+          <a href="https://www2.gov.bc.ca/gov/content/industry/crown-land-water/land-use-planning">
+            government’s land use planning program homepage
+          </a>
+          to learn more about provincial land use planning policy and guidance, completed provincial land use plans and
+          legal direction, and spatial data.
         </p>
         <h2>Land Use Planning Process</h2>
         <p>
           Land use planning takes a phased approach that progressively builds recommended
           options, collecting and conducting analysis on key values identified and engaged
-           on throughout the planning process. A land use planning process can be separated
-            into four broad stages.
+          on throughout the planning process. A land use planning process can be separated
+          into four broad stages.
         </p>
-        <img src="assets/images/lup-phases.png" alt="The Land Use Planning Phases, in order: the
-        Pre-Planning and Initiation stage, the Plan Development stage, the Plan Evaluation and
-        Approval stage, and the Implementation and Monitoring stage." />
-        <h2>Pre-Planning and Initiation Stage</h2>
+        <img src="assets/images/lup-phases.png" alt="The Land Use Planning Phases, in order: 
```

Comment:
A couple of small notes on the image. First, the image width is about twice the width of the full-width container, so you may want to consider downscaling it to improve load times. Also, it looks like the previous image was centred properly but the new image has more space on the bottom. It might be okay, adjust padding or crop image at your own discretion.